apps/zerg/agent/Makefile

```
@@ -0,0 +1,64 @@
+.PHONY: all format lint test tests test_watch integration_tests docker_tests help extended_tests
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)